.github/workflows/run-k8s-tests-on-zvsi.yaml

```diff
@@ -78,3 +78,62 @@ jobs:
         run: |
           bash tests/integration/kubernetes/gha-run.sh cleanup-zvsi || true
           ${HOME}/script/post_action.sh ubuntu-2204
+
+  run-k8s-tests-coco-nontee:
+    strategy:
+      fail-fast: false
+      matrix:
+        vmm:
+          - qemu
+        snapshotter:
+          - nydus
+        pull-type:
+          - guest-pull
+    runs-on: s390x-large
+    env:
+      DOCKER_REGISTRY: ${{ inputs.registry }}
+      DOCKER_REPO: ${{ inputs.repo }}
+      DOCKER_TAG: ${{ inputs.tag }}
+      GH_PR_NUMBER: ${{ inputs.pr-number }}
+      KATA_HOST_OS: ${{ matrix.host_os }}
```

Comment:
There doesn't seem to be a `host_os` matrix parameter, so should this `env` be set to `ubuntu` (I guess?) directly?

Reply:
oh, good catch. I will update the PR. Thanks!